torchqdynamics/solver_utils.py

```diff
@@ -28,3 +28,18 @@ def inv_sqrtm(mat: torch.Tensor) -> torch.Tensor:
     """
     vals, vecs = torch.linalg.eigh(mat)
     return vecs @ torch.linalg.solve(vecs, torch.diag(vals**(-0.5)), left=False)
+
+
+def bexpect(operators: torch.Tensor, state: torch.Tensor) -> torch.Tensor:
+    """Compute the expectation values of many operators on a quantum state or
+    density matrix. The method is batchable over the operators and the state.
+
+    Args:
+        operators: tensor of shape (m, n, n)
```

Comment:
Can we use `b` instead of `m` for the batch size? This way we keep `m` for `(n, m)` for kets and bras.